opts/opts.go

```diff
@@ -17,6 +17,7 @@ var (
 	alphaRegexp       = regexp.MustCompile(`[a-zA-Z]`)
 	domainRegexp      = regexp.MustCompile(`^(:?(:?[a-zA-Z0-9]|(:?[a-zA-Z0-9][a-zA-Z0-9\-]*[a-zA-Z0-9]))(:?\.(:?[a-zA-Z0-9]|(:?[a-zA-Z0-9][a-zA-Z0-9\-]*[a-zA-Z0-9])))*)\.?\s*$`)
 	DefaultHTTPHost   = "127.0.0.1"            // Default HTTP Host used if only port is provided to -H flag e.g. docker -d -H tcp://:8080
+	DefaultHTTPPort   = 2375                   // Default HTTP Port
```

Comment:
@tiborvass did you mix them up, or am I mis-interpreting your comment? 2375 is HTTP and 2376 is HTTPS; http://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml?search=docker